### PR TITLE
OSDOCS-11708:updates 3-4 planning doc for SDN removal

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.16

Issue:
https://issues.redhat.com/browse/OSDOCS-11708

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: migrating_from_ocp_3_to_4/planning-migration-3-4.adoc [hunk 1]
```diff
@@ -166,6 +166,23 @@ In {product-title} 3.11, OpenShift SDN was the default networking plugin in Red
 
 For information on migrating to OVN-Kubernetes from OpenShift SDN, see xref:../networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc#migrate-from-openshift-sdn[Migrating from the OpenShift SDN network plugin].
 
+[WARNING]
+====
```
Comment (zhaozhanqi) [inline migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:170]: hmm, do we need to mentioned this in this part openshift migration 3 to 4?
what's the meaning add this for migration 3 to 4?

File: migrating_from_ocp_3_to_4/planning-migration-3-4.adoc [hunk 2]
```diff
@@ -166,6 +166,23 @@ In {product-title} 3.11, OpenShift SDN was the default networking plugin in Red
...
+It is not possible to upgrade a cluster to {product-title} 4.17 if it is using the OpenShift SDN network plugin. You must migrate to the OVN-Kubernetes plugin before upgrading to {product-title} 4.17.
+====
+
+For information on OVN-Kubernetes features that are similar to features in the OpenShift SDN plugin see:
```
Comment (danwinship) [inline migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:174]: In the 4.16 docs, this is already covered by the migration docs you linked to above, so I'm not sure you need to copy this part

File: migrating_from_ocp_3_to_4/planning-migration-3-4.adoc [hunk 3]
```diff
@@ -166,6 +166,23 @@ In {product-title} 3.11, OpenShift SDN was the default networking plugin in Red
...
+
+* xref:../networking/ovn_kubernetes_network_provider/configuring-egress-ips-ovn.adoc#configuring-egress-ips[Configuring an egress IP address]
+
+* xref:../networking/network_security/egress_firewall/configuring-egress-firewall-ovn.adoc#configuring-egress-firewall-ovn[Configuring an egress firewall for a project]
+
+* xref:../networking/ovn_kubernetes_network_provider/enabling-multicast.adoc#enabling-multicast[Enabling multicast for a project]
+
+* xref:../networking/ovn_kubernetes_network_provider/deploying-egress-router-ovn-redirection.adoc#deploying-egress-router-ovn-redirection[Deploying an egress router pod in redirect mode]
+
+* xref:../networking/network_security/network_policy/multitenant-network-policy.adoc#multitenant-network-policy[Configuring multitenant isolation with network policy]
```
Comment (danwinship) [inline migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:184]: (and this part in particular is already linked from the "Network isolation mode" section earlier in this doc.)

**Review by danwinship — COMMENTED** [review]
The [WARNING] LGTM. Not sure you need the rest?

File: migrating_from_ocp_3_to_4/planning-migration-3-4.adoc (hunk 1 above)
Comment (JoeAldinger) [reply migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:170]: My concern is I don't want customers to be surprised by this. That is why I'd like to add this.

File: migrating_from_ocp_3_to_4/planning-migration-3-4.adoc (hunk 2 above)
Comment (JoeAldinger) [reply migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:174]: Good point, Dan. I'll remove this, but I think we should still keep the warning. What do you think? @danwinship

Comment (danwinship) [reply migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:174]: yes, exactly. keep the warning, lose the rest

File: migrating_from_ocp_3_to_4/planning-migration-3-4.adoc (hunk 1 above)
Comment (JoeAldinger) [reply migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:170]: @zhaozhanqi does the update work for you?

Comment (zhaozhanqi) [reply migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:170]: Hmm,  I remembered cluster migration from openshift 3.x to 4.x is not supported.  so I'm not sure if this is needed

**Comment by zhaozhanqi** [comment]
LGTM

**Pull request merged**